.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,35 @@
+name: Benchmark GTSFM on select datasets
+on: [pull_request]
+jobs:
+  door-dataset-sift:
+    name: Run GTSFM on the door dataset using SIFT front-end
```

Comment:
looks good. can we run the skydio-8 here also, and skydio-32? i guess we can put those in G Drive or upload as files to a separate Github location (not checked into this repo, though)

Reply:
marking as resolved, thanks for adding this